cni-metrics-helper/README.md

```diff
@@ -0,0 +1,75 @@
+# cni-metrics-helper
+
+The `cni-metrics-helper` is a tool that can be used to scrape the aggregated ENIs and IPs information at the cluster 
```

Comment:
I think they should have '

Reply:
Or maybe aggregated ENI and IP information. Can't grammar. Line 24 too in troubleshooting.md